### ISSUE TITLE
Transfer the `llvm` npm name

### ISSUE THREAD

**Issue by espadrine** [kevinmehall/node-llvm#3]
I can't help but notice you stopped working on this project.

I also noticed that it is out of date, and that it has been forked under `llvm-x` on npm. Those that forked it added a mere two commits on top of your work, the most significant (although not that significant) being https://github.com/guangwong/node-llvm-x/commit/72d727024be41bb7e20241120086f2a6e058feb5.

Would you consider either pulling this commit and submitting a new version, or transfering the `llvm` name to them?

**Comment by kevinmehall** [comment]
You're correct that this library is no longer maintained. My project that it was originally intended for has been rewritten in Rust.

You should also look at [colesbury's fork](https://github.com/colesbury/node-llvm/commits/master), who's fixed several bugs and added more bindings. I'm happy to add owners to the `npm` package if someone is interested in maintaining the library. 

@colesbury, @guangwong, any interest?

**Comment by tiye** [comment]
I found @guangwong released his package as https://www.npmjs.com/package/llvm-x ... sounds strange.
